test/e2e/proxy_publicviewer_test.go

```diff
@@ -231,16 +235,24 @@ func TestProxyPublicViewer(t *testing.T) {
 		require.NoError(t, err)
 		require.Empty(t, sbs)
 
-		testCases := map[string]bool{
-			"user is not banned": false,
-			"user is banned":     true,
+		testCases := []TestCase{
```

Comment:
You can use the following format instead:
```
	tests := map[string]struct {
		Banned         bool
		ExpectedError          string
	}{
		"user is not banned": {
			Banned: false,
			ExpectedError: "invalid workspace request",
		},
	}
```

Reply:
done in [cfa49c2](https://github.com/codeready-toolchain/toolchain-e2e/pull/1043/commits/cfa49c266969389c217352754504a3e450b8a1c6)